ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.17-nightly-x86.yaml

```diff
@@ -119,6 +119,18 @@ tests:
     - chain: openshift-qe-node-density-heavy
     workflow: openshift-qe-installer-aws
   timeout: 5h0m0s
+- as: node-density-heavy-ipsec-24nodes
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      ZONES_COUNT: "3"
+    test:
+    - chain: openshift-qe-node-density-heavy
```

Comment:
you can try first adding this step [workers-scale](https://github.com/openshift/release/blob/e2f50d07441c5391bfc16c1b1884f5ae3fc3e489/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__ibmcloud-4.18-nightly-x86.yaml#L65) with WORKERS=24 and COMPUTE_NODE_REPLICAS=3 to start for the cluster to be able to create. The network is waiting for cluster creation probably becuase it adds a pod for every node and theres too many nodes to start